src/main/java/querqy/elasticsearch/rewriterstore/PutRewriterResponse.java

```diff
@@ -32,7 +33,6 @@ public void writeTo(final StreamOutput out) throws IOException {
         reloadResponse.writeTo(out);
     }
 
-    @Override
```

Comment:
See related comment in src/main/java/querqy/elasticsearch/rewriterstore/DeleteRewriterResponse.java